contracts/test/TestERC721Enumerable.sol

```diff
@@ -28,9 +28,10 @@ contract TestERC721Enumerable is ERC721, ERC721Enumerable, Ownable {
     function _beforeTokenTransfer(
         address from,
         address to,
-        uint256 tokenId
+        uint256 firstTokenId,
+        uint256 batchSize
     ) internal override(ERC721, ERC721Enumerable) {
-        super._beforeTokenTransfer(from, to, tokenId);
+        super._beforeTokenTransfer(from, to, firstTokenId, batchSize);
```

Comment:
@sanderpick I'm assuming this change is required to do the OZ upgrade?

Reply:
Yep!